### PR TITLE
Update operations-configuring_check_plugins.xml

### PR DESCRIPTION
**Pull request by yarunachalam**
Line 74 ansible example replaced with new example as per defect https://bugzilla.suse.com/show_bug.cgi?id=1098302

### CODE REVIEW

File: xml/operations-configuring_check_plugins.xml [hunk 1]
```diff
@@ -71,14 +71,12 @@
...
  </para>
-<screen>- name: Run Monasca Agent detection plugin for Cassandra
-  monasca_agent_plugin:
-    name: "Cassandra"
-    args:
-      user: {{ FND_VDB.vars.monitor_user_name }}
-      password: {{ FND_VDB.vars.monitor_user_password }}
-      timeout: 10
-  no_log: True</screen>
+<screen>tasks:
```
Comment (jodavis-suse) [inline xml/operations-configuring_check_plugins.xml]: My cut and paste issue - you don't need to include the tasks: line.
Scott is going to look at the SampleCassandraPlugin code, so you might hold merging this until that change is also included.

**Comment by csymons-suse** [comment]
does this need to be tested before it can be merged?

**Comment by sgrasley** [comment]
Just example code, should be good without testing. 

-Scott

**Comment by jodavis-suse** [comment]
Please do remote the "tasks:" before merging.

**Pull request merged**